apps/fm/js/fm.js

```diff
@@ -883,6 +890,12 @@ document.addEventListener('DOMLocalized', function onDOMLocalized(e) {
   window.performance.mark('navigationLoaded');
   document.removeEventListener('DOMLocalized', onDOMLocalized);
 
+  // localize MHz
+  document.l10n.format('mhz').then(str => {
```

Comment:
Will this automatically re-localize if the locale is changed after the app is launched?